packages/cli-repl/src/cli-repl.ts

```diff
@@ -795,6 +798,76 @@ export class CliRepl implements MongoshIOProvider {
     }
   }
 
+  verifyPlatformSupport(): void {
+    if (this.cliOptions.quiet) {
+      return;
+    }
+
+    // Typings for process.getReport haven't been updated
+    // (https://github.com/DefinitelyTyped/DefinitelyTyped/issues/40140)
+    const processReport = process.report?.getReport() as unknown as
+      | {
+          header: {
+            glibcVersionRuntime: string;
```

Comment:
```suggestion
            glibcVersionRuntime?: string;
```

(not present on non-Linux platforms)